training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/requirements.txt

```diff
@@ -0,0 +1,5 @@
+mlflow==2.3.0
```

Comment:
are these exclusive dependency list or a subset?

Reply:
Yea, these are exclusive dependency list for required packages in deployment.
Packages MMCV, mmdet can not be installed properly through PIP. Hence, they are installed in Model load script using OpenMIM package.